### DISCUSSION TITLE
Beatmap difficulty parameters should be visible when selecting or customizing mods

### DISCUSSION THREAD

**Discussion by Dimmitsaras** [ppy/osu#23621] [Ideas] [answered]
Preface: I am using 1080p 100% menu UI scaling for reference

Applying mods can change the details of a beatmap, for example circle size, BPM or star rating. These update on the background as soon as a selection or adjustment is made, but this is not taken advantage of because the mod menu obstructs view.

A common scenario would be adjusting DT/HT to reach a specific BPM, or getting at just the next integer star rating. Currently the user has to exit out of the mod menu to check how their changes affect the map parameters

The map parameters in question:

![image](https://github.com/ppy/osu/assets/35996393/eb84663e-7a9c-457b-a7cb-0bde7af4fbf5)

Opening the mod menu obstructs the parameters:

![image](https://github.com/ppy/osu/assets/35996393/2f3b6e60-906c-4f89-8589-d507bb0f2793)

Case of micro adjusting DT:

![image](https://github.com/ppy/osu/assets/35996393/d251d2ba-f8cd-4b0a-a9de-20289726565e)

Not exactly sure how to fix this, but the "simplest" way is to pad the mod menu to the right and move it a layer below the map tab.

**Comment by monochrome22** [comment, marked as answer]
Well, it's a duplicate of this: https://github.com/ppy/osu/discussions/18189.
Some discussion in another thread, including peppy's answer regarding displaying difficulty, which was also marked duplicate (confusing situation): https://github.com/ppy/osu/discussions/18172#discussioncomment-2713437.
Here it was mentioned that displaying post edit BPM for DT is planned https://github.com/ppy/osu/discussions/22616#discussioncomment-4948833.
